frappe/core/doctype/data_import/importer.py

```diff
@@ -149,8 +149,8 @@ def import_data(self):
 
 					if self.console:
 						update_progress_bar(
-							f"Importing {total_payload_count} records",
-							current_index,
+							f"Importing ({self.import_type.split()[0]}) for doctype {self.doctype}: {total_payload_count} records",
```

Comment:
```suggestion
							f"Importing {self.doctype}: {total_payload_count} records",
```

This won't be a valid English sentence? 


"importing Insert New Records for doctype" ?


Let's assume user knows what the type is because they just passed it as argument.